src/github/types.rs

```diff
@@ -35,15 +35,15 @@ pub struct Comment {
     pub created_at: DateTime<Utc>,
 }
 
-#[derive(Clone, Deserialize)]
+#[derive(Deserialize)]
```

Comment:
why get rid of the derivation? I've generally found that I like having as many applicable derivations as possible for future use, and then you can just let the compiler remove the dead code if they happen to not be used. I suppose it doesn't really matter since we are the only ones consuming this type so we can just change it if we happen to need it later.

Reply:
I was on the fence about this. On one hand, it's nice to have all of those available when you need them. On the other, I want to force people to consider if they _actually_ need to clone. I'd like to leave it the way it is for now and then maybe follow up with a uniform set of derivations.